doc/services/logging/index.rst

```diff
@@ -309,20 +315,20 @@ Following snippet shows how logging can be processed in simple forever loop.
 
    void main(void)
    {
-   	log_init();
+   	LOG_INIT();
+        /* If multithreading is enabled provide thread id to the logging. */
+        log_thread_set(k_current_get());
```

Comment:
Indentation? Possibly spaces vs tabs.

Reply:
Seems to appear fine here, 
https://builds.zephyrproject.io/zephyr/pr/50741/docs/services/logging/index.html#controlling-the-logging

Reply:
Seems to appear fine here,
https://builds.zephyrproject.io/zephyr/pr/50741/docs/services/logging/index.html#controlling-the-logging